src/Raven.Studio/typescript/commands/database/query/queryCommand.ts

```diff
@@ -98,22 +108,29 @@ f();
 `;
         let parameters = eval(parametersJs);
         const rql = queryText.substring(match.index);
-        return [parameters, rql];
+        return [JSON.parse(parameters), rql];
```

Comment:
why we parse here something that was just stringified? 

Can't we return object from getPayload function?